dbt/adapters/postgres.py

```diff
@@ -213,19 +214,23 @@ def reset_csv_table(cls, profile, schema, table_name, agate_table,
 
     @classmethod
     def load_csv_rows(cls, profile, schema, table_name, agate_table):
-        bindings = []
-        placeholders = []
         cols_sql = ", ".join(c for c in agate_table.column_names)
 
-        for row in agate_table.rows:
-            bindings += row
-            placeholders.append("({})".format(
-                ", ".join("%s" for _ in agate_table.column_names)))
+        for chunk in chunks(agate_table.rows, 10000):
+            bindings = []
+            placeholders = []
 
-        sql = ('insert into {}.{} ({}) values {}'
-               .format(cls.quote(schema),
-                       cls.quote(table_name),
-                       cols_sql,
-                       ",\n".join(placeholders)))
+            for row in chunk:
+                bindings += row
+                placeholders.append("({})".format(
+                    ", ".join("%s" for _ in agate_table.column_names)))
 
-        cls.add_query(profile, sql, bindings=bindings, abridge_sql_log=True)
+                sql = ('insert into {}.{} ({}) values {}'
+                       .format(cls.quote(schema),
+                               cls.quote(table_name),
+                               cols_sql,
+                               ",\n".join(placeholders)))
+
+            cls.add_query(profile, sql,
+                          bindings=bindings,
```

Comment:
@cmcarthur is this correct? It seems to me that the second chunk would get the bindings from the first chunk?

Reply:
bindings is reset inside the outer loop